qdrant-landing/content/documentation/guides/running-with-GPU.md

```diff
@@ -0,0 +1,204 @@
+2025-01-13T11:58:29.124118Z  INFO gpu::instance: Foung GPU device: llvmpipe (LLVM 15.0.7, 256 bits)    
+2025-01-13T11:58:29.124138Z  INFO gpu::device: Create GPU device NVIDIA GeForce RTX 3090    
+```
+
+In this example logs you may see that 2 devices were found: RTX 3090 and llvmpipe (a CPU-emulated GPU which is included into docker image). And later you may see that only RTX was initialized.
+
+That’s it. In a basic scenario, there is nothing to do more.
+
+### Troubleshoots
+
+If your GPU is not detected in docker, check first your driver and `nvidia-container-toolkit` versions.
+They both should be latest.
+
+To check that Vulkan API is visible in a docker container, try to run this command:
+
+```
+docker run --rm --gpus=all qdrant/qdrant:gpu-nvidia-latest vulkaninfo --summary
+```
+
+You may find an error message with an explanation of why the NVidia device is not visible.
+If your NVidia GPU is not visible in Docker, Docker image cannot use libGLX_nvidia.so.0 on your host. Example of an error message:
+
+```
+ERROR: [Loader Message] Code 0 : loader_scanned_icd_add: Could not get `vkCreateInstance` via `vk_icdGetInstanceProcAddr` for ICD libGLX_nvidia.so.0
+WARNING: [Loader Message] Code 0 : terminator_CreateInstance: Failed to CreateInstance in ICD 0. Skipping ICD.
+```
+
+To fix it, try to change the config:
+
+```
+sudo nano /etc/nvidia-container-runtime/config.toml
+```
+
+Set `no-cgroups=false`, save config, and restart docker
+
+```
+sudo systemctl restart docker
+```
+
+## AMD GPUs
+
+AMD docker images have the same tag as qdant but with `gpu-amd` suffix, for example `qdrant/qdrant:v1.13.0-gpu-amd`.
+This docker image contains all required dependencies to run Qdrant with GPU support.
+
+To enable GPU for docker, you need an additional `--device /dev/kfd --device /dev/dri` flags. To enable GPU for Qdrant you need to set enable flag. Example of starting Qdrant with AMD GPU support:
+
+```bash
+# `--device /dev/kfd --device /dev/dri` flags say to Docker that we want to use GPUs.
+# `-e QDRANT__GPU__indexing=1` flag says to QDrant that we want to use GPUs for indexing.
+docker run \
+	--rm \
+	--device /dev/kfd --device /dev/dri \
+	-e QDRANT__log_level=debug \
+	-p 6333:6333 \
+	-p 6334:6334 \
+	-e QDRANT__GPU__indexing=1 \
+	qdrant/qdrant:gpu-amd-latest
+```
+
+Check logs to confirm GPU initialization. Example log output:
+
+```text
+2025-01-10T11:56:55.926466Z  INFO gpu::instance: Foung GPU device: AMD Radeon Graphics (RADV GFX1103_R1)
+2025-01-10T11:56:55.926485Z  INFO gpu::instance: Foung GPU device: llvmpipe (LLVM 17.0.6, 256 bits) 
+2025-01-10T11:56:55.926504Z  INFO gpu::device: Create GPU device AMD Radeon Graphics (RADV GFX1103_R1)
+```
+
+This concludes the setup. In a basic scenario, you won't need to configure anything else.
+
+
+## Known limitations
+
+* **Platform Support:** Only Linux x86_64 is supported. Windows, macOS, ARM, and other platforms are not supported.
```

Comment:
Linux x64 is a limitation of docker image. Is user run from binary, it doesn't matter. I'm working under windows as well